assets/queries/cloudFormation/rds_public_virtual_interface/test/positive.yaml

```diff
@@ -0,0 +1,92 @@
+#this is a problematic code where the query should report a result(s)
+Resources:
+  DBEC2SecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Open database for access
+      SecurityGroupIngress:
+      - IpProtocol: tcp
+        FromPort: 80
+        ToPort: 80
+        CidrIp: 3.5.140.0/22
+      SecurityGroupEgress:
```

Comment:
`E2521 Property VpcId should exist with SecurityGroupEgress for Resources/DBEC2SecurityGroup/Properties`